ldes-server-port-admin-rest/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/admin/rest/controllers/AdminViewsRestController.java

```diff
@@ -15,14 +16,16 @@
 @RequestMapping("/admin/api/v1")
 public class AdminViewsRestController {
 	private final LdesConfigModelService service;
+	// TODO use viewService instead of LdesConfigModelService
```

Comment:
TODO can be removed?

Reply:
currently, the LdesConfigModelService is still used. Will delete this as soon as the dependency on LdesConfigModelService is removed.